gdpr/vendorlist-fetching.go

```diff
@@ -81,35 +86,30 @@ func makeVendorListNotFoundError(vendorListVersion uint16) error {
 }
 
 // preloadCache saves all the known versions of the vendor list for future use.
-func preloadCache(ctx context.Context, client *http.Client, urlMaker func(uint16, uint8) string, saver saveVendors, tcfSpecVersion uint8) {
-	latestVersion := saveOne(ctx, client, urlMaker(0, tcfSpecVersion), saver, tcfSpecVersion)
+func preloadCache(ctx context.Context, client *http.Client, urlMaker func(uint16) string, saver saveVendors) {
+	latestVersion := saveOne(ctx, client, urlMaker(0), saver)
+	firstVersion := uint16(2) // The GVL for TCF2 has no vendors defined and is very unlikely to be used. Don't load it.
```

Comment:
Incomplete comment. It is the first version of the GVL for TCF2 that has no vendors, not all TCF2 GVLs. :)